### PR TITLE
[Example] Add style transfer example

### PR DESCRIPTION
**Pull request by miaobin**
Add style transfer example:
WebGL - support
Wasm - conv2dTranspose not supported

Need to define the following ops:
SUB, DIV, POW, PAD, MEAN, TRANSPOSE_CONV_2D.
@belem @huningxin @BruceDai PTAL, Thanks!

### CODE REVIEW

**Comment by miaobin** [comment]
BTW, the pre-trained models can be converted to tflite format now. And we need to support another operator - SquaredDifference.

File: examples/style_transfer/model/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+## Training and Conversion
```
Comment (ibelem) [inline examples/style_transfer/model/README.md:1]: @miaobin please upload trained/converted models to google drive as well with shared permission.

File: examples/style_transfer/index.html [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+                          <li><img class="image" id="sunflowers-bew" src='./img/Sunflowers-Bew.jpg'  title="“Sunflowers” (1889)" /></li>
+                          <li><img class="image" id="red-vineyards" src='./img/Red-Vineyards.jpg' title="The Red Vineyard"  /></li>
+                          <li><img class="image" id="sien_with_a_cigar" src='./img/sien_with_a_cigar.jpg'  title="Sien with a cigar" /></li>
+                          <li><img class="image" id="soup-distribution" src='./img/Soup-Distribution.jpg'  title="Soup Distribution in a Public Soup Kitchen" /></li>
+                          <li><img class="image" id="wheatfield_with_crows" src='./img/Wheatfield_with_Crows.jpg'  title="Wheatfield with Crows" /></li>
+                          <li><img class="image" id="la-campesinos" src='./img/la-campesinos.jpg'  title="Rest from Work" /></li>
+                        </ul>
+                      </div>
+                    </div>
+                  </div>
```
Comment (ibelem) [inline examples/style_transfer/index.html:247]: Please indicate on page that which style is using when inferencing.

**Comment by ibelem** [comment]
@ibelem the models have been uploaded to Amazon S3.

**Comment by ibelem** [comment]
LGTM, thanks a lot @miaobin ! Some UI changes will be in new patches.

**Pull request merged**